go/vt/vtgate/schema/tracker.go

```diff
@@ -136,6 +139,10 @@ func (t *Tracker) initKeyspace(th *discovery.TabletHealth) error {
 	err := t.LoadKeyspace(th.Conn, th.Target)
 	if err != nil {
 		log.Warningf("Unable to add the %s keyspace to the schema tracker: %v", th.Target.Keyspace, err)
+		code := vterrors.Code(err)
+		if code == vtrpcpb.Code_UNAUTHENTICATED || code == vtrpcpb.Code_PERMISSION_DENIED {
+			log.Warning("Table ACL might be enabled, --schema_change_signal_user needs to be passed to VTGate for schema tracking to work. More details on vitess.io")
```

Comment:
Instead of `More details on vitess.io`, we should say `See <link>` and link to the docs page.
Of course we'll need to update that link in every release 😓

Reply:
for the same reason I did not give full link

Reply:
We should really start using error codes. Makes it very easy to search for the more info on specific errors, and is easier to manage on both ends.

This is how Oracle does it:
https://docs.oracle.com/cd/E11882_01/server.112/e17766/toc.htm

Reply:
how about `Check 'schema tracking' docs on vitess.io`

Reply:
I agree, we can take this up for 15.0 but I like to avoid putting link in the error message inside the code.
should I go ahead with this change?

Reply:
sure, this should work as a stop-gap measure, IMHO. Let's leave it to @deepthi to make a call on it

Reply:
that's fine for now.